lambdas/migration/handler.py

```
@@ -0,0 +1,93 @@
+    results = []
+
+    try:
+        for migration in migrations:
+            filename = migration.name
+            logger.info("Running migration: %s", filename)
+            try:
+                run_migration(conn, migration)
+                results.append({"file": filename, "status": "success"})
+                logger.info("Completed: %s", filename)
+            except Exception as e:
+                logger.exception("Failed migration %s: %s", filename, e)
```

Ruff (0.14.13)

Redundant exception object included in `logging.exception` call

(TRY401)